bin/shaders/GSdx.fx

```diff
@@ -2031,6 +2084,304 @@ float4 BorderPass(float4 colorInput, float2 tex)
 }
 
 /*------------------------------------------------------------------------------
+                           [LOTTES CRT CODE SECTION]
+------------------------------------------------------------------------------*/
+
+#if LOTTES_CRT == 1
+float ToLinear1(float c)
+{
+    c = saturate(c);
+    return(c <= 0.04045) ? c / 12.92 : pow((c + 0.055) / 1.055, 2.4);
+}
+
+float3 ToLinear(float3 c)
+{
+    return float3(ToLinear1(c.r), ToLinear1(c.g), ToLinear1(c.b));
+}
+
+float ToSrgb1(float c)
+{
+    c = saturate(c);
+    return(c < 0.0031308 ? c * 12.92 : 1.055 * pow(c, 0.41666) -0.055);
+}
+
+float3 ToSrgb(float3 c)
+{
+    return float3(ToSrgb1(c.r), ToSrgb1(c.g), ToSrgb1(c.b));
+}
+
+float3 Fetch(float2 pos, float2 off)
+{
+    float2 crtRes = float2(CRTSizeX, CRTSizeY);
+    float2 res = (crtRes * ResolutionScale);
+    pos = floor(pos * res + off) / res;
```

Comment:
`floor()` seems to shift the image slightly to the bottom right (most visible at native res). Would not rounding here be better?